templates/repo/issue/view_content.tmpl

```diff
@@ -27,7 +27,7 @@
 				</a>
 			{{end}}
 				<div class="content comment-container">
-					<div class="ui top attached header comment-header df ac sb">
+					<div class="ui top attached header comment-header df ac sb" role="heading" aria-level="3">
```

Comment:
Hmm, is that really a heading?
I mean, it doesn't have any title-like attributes, and doesn't behave exactly like a title…

Reply:
Isn't the objective just to use Aria article like accessibility for readers?

Reply:
> Hmm, is that really a heading? I mean, it doesn't have any title-like attributes, and doesn't behave exactly like a title…

The whole container functions like a comment head, but change the container to a H3 it has a deep impact in actual behavior. That's why this proposition does an aria fix.

Reply:
Exactly, @fsologureng is totally right. That's btw how it's done on Github: the whole `username commented...` thing is wrapped into a heading.  
The `article` is good but not enough for navigation. Navigation by headings is in the blood of most blind web users, and a page that lacks headings is extremely inconvenient to navigate through. If you have an `article` with the comment, you have to: go to the article, then arrow up, i.e., back to see the author, then arrow down again past the author to read the comment text now. This is super inconvenient as it requires moving back and forth in the same text. Please note that blind users A) don't use mouse, only keyboard, and B) we navigate in a _linear_ way, we always arrow up and down (we also use the *Tab* key, but it's out of this scope).

Reply:
How important it is that designers understand this.